locals.tf

```diff
@@ -1,7 +1,6 @@
 locals {
   create_task_definition = var.deployment_controller_type != "EXTERNAL" && var.create_task_definition ? true : false
   create_lb_sg           = var.create_load_balancer && length(var.lb_security_group_ingress) > 0 ? true : false
-  legible_lb_type        = title(var.load_balancer_type) == "Application" || title(var.load_balancer_type) == "Network" ? true : false
```

Comment:
Is the `? true : false` really necessary here? wouldn't `title(var.load_balancer_type) == "Application" || title(var.load_balancer_type) == "Network"` by itself be a boolean already? not sure if Terraform supports this, though.

Reply:
Actually not necessary... I removed that line since it didn't have any effect on the module.